app/models/manageiq/providers/amazon/network_manager/refresh_parser.rb

```diff
@@ -70,6 +79,49 @@ def get_outbound_firewall_rules(sg)
     sg.ip_permissions_egress.collect { |perm| parse_firewall_rule(perm, "outbound") }.flatten
   end
 
+  def get_load_balancers
+    process_collection(load_balancers, :load_balancers) { |lb| parse_load_balancer(lb) }
+  end
+
+  def get_load_balancer_pools
+    process_collection(load_balancers, :load_balancer_pools) { |lb| parse_load_balancer_pool(lb) }
+    get_load_balancer_pool_members
+  end
+
+  def get_load_balancer_pool_members
+    @data[:load_balancer_pool_members] = []
+
+    load_balancers.each do |lb|
+      new_lb = @data_index.fetch_path(:load_balancer_pools, lb.load_balancer_name)
+      load_balancer_pool_members = lb.instances.collect { |m| parse_load_balancer_pool_member(m) }
+      load_balancer_pool_members.each do |x|
```

Comment:
minor, but variable like `x` always scream to me :smile: 
I guess x is really a member? so how about  hitting F6 in RubyMine and change it to member?